quickstart/README.md

```diff
@@ -50,22 +52,19 @@ This quickstart makes a few assumptions about the target operating system and is
    # Create a jwt access token for use with the apis.
    ACCESS_TOKEN=$(gen_access_token)
    # If you're curious about that token, you can safely copy and paste it into https://jwt.io to learn more.
-   # Use curl to confirm that everything is working
-   curl --cacert cacert.pem -H "Authorization: Bearer $ACCESS_TOKEN" https://foobar.openchami.cluster/hsm/v2/State/Components
+
+1. Confirm services are up and running   
+   Check you can access SMD. Component endpoint is unprotected
+   ```bash
+   curl --cacert cacert.pem https://foobar.openchami.cluster:8443/hsm/v2/State/Components
    # This should respond with an empty set of Components: {"Components":[]}
    ```
-1. Create a token that can be used by the dnsmasq-loader which reads from smd.  This activates our automatic dns/dhcp system.  The command automatically adds it to .env
-   ```bash
-    echo "DNSMASQ_ACCESS_TOKEN=$(gen_access_token)" >> .env
-    ```
-1. Use docker-compose to bring up your dnsmasq contianers.  The only difference between this command and the one above is the addition of the `dnsmasq.yml` file.  Docker compose needs to know about all the files to follow dependencies.
+   Check if you can read Redfish endpoints. This endpoint is protected
    ```bash
-   docker compose -f base.yml -f postgres.yml -f jwt-security.yml -f haproxy-api-gateway.yml -f openchami-svcs.yml -f autocert.yml -f dnsmasq.yml up -d
+   curl --cacert cacert.pem -H "Authorization: Bearer $ACCESS_TOKEN" https://foobar.openchami.cluster:8443/hsm/v2/Inventory/RedfishEndpoints
```

Comment:
I guess we are sticking with port 8443 for now then. It might be worth pointing out that this is running on a non-standard port.